js/components/generate/GenerateForm.js

```diff
@@ -296,91 +295,92 @@ export default class GenerateForm extends React.Component {
+                <div className="form-section">
+                  <h2 id="filter-title" className="section-title">Filters</h2>
+
+                  <label htmlFor="departments">Departments</label>
+                  <Field
+                    id="departments"
+                    name="departments"
+                    type="text"
+                    placeholder="Enter 3 letter dept. codes, sep. by commas"
+                  />
+                  <div className="error-container">
+                    <ErrorMessage
+                      className="error-message"
+                      name="departments"
+                      component="div"
+                    />
+                  </div>
+
+                  <label htmlFor="taken">Hide courses</label>
+                  <Field
+                    id="taken"
+                    name="taken"
+                    type="text"
+                    placeholder="E.g., CSC207H1, CSC236H1"
+                  />
+                  <div className="error-container">
+                    <ErrorMessage
+                      className="error-message"
+                      name="taken"
+                      component="div"
+                    />
+                  </div>
+
+                  <label htmlFor="maxDepth">Prereq chain depth (0 shows all)</label>
+                  <Field id="maxDepth" name="maxDepth" type="number" min="0" step="1" />
+
+                  {/* <label htmlFor="location">Campus</label>
+                  <Field id="location" name="location" as="select" multiple
+                    style={{ verticalAlign: 'text-top', marginLeft: '1em', marginBottom: '1em', color: 'black' }}>
+                    <option value="utsg">St. George</option>
+                    <option value="utm">Mississauga</option>
+                    <option value="utsc">Scarborough</option>
+                  </Field>
+
+                  <p>
+                    <label htmlFor="includeRaws">Include non-course prerequisites</label>
+                    <Field id="includeRaws" name="includeRaws" type="checkbox"
+                      style={{ marginLeft: '1em', verticalAlign: 'middle' }}
+                    />
+                  </p>
+
+                  <label htmlFor="includeGrades">Include grade-based prerequisites</label>
+                  <Field id="includeGrades" name="includeGrades" type="checkbox"
+                    style={{ 'margin-left': '1em', 'vertical-align': 'middle' }} /> */}
+                </div>
 
                 <div
                   style={{
-                    marginTop: "1em",
                     width: "100%",
                     display: "flex",
                     justifyContent: "center",
                   }}
                 >
                   <button id="submit" type="submit">
-                    Generate Graph
+                    <img id="generate-icon" src="/static/res/ico/generate.png" />
```

Comment:
Overall for static assets we should always provide a source for the asset (and not just add to the repo).

In this case, let's use font awesome (which is also what MarkUs uses). I think the closest one to the image you have is https://fontawesome.com/icons/wand-sparkles?f=classic&s=solid. See also https://docs.fontawesome.com/v5/web/use-with/react for information about using font awesome with React.